joern-cli/frontends/csharpsrc2cpg/src/main/scala/io/joern/csharpsrc2cpg/datastructures/CSharpScope.scala

```diff
@@ -10,7 +10,7 @@ import scala.collection.mutable
 
 class CSharpScope(typeMap: TypeMap) extends Scope[String, DeclarationNew, ScopeType] {
 
-  private val typesInScope = mutable.Set.empty[CSharpType]
+  private val typesInScope = mutable.Set.empty[CSharpType].addAll(typeMap.findGlobalTypes)
```

Comment:
What are these global types? Since the effect here is that global types from other files are visible in this file, is that intended?

Reply:
Yes, classes without a parent namespace are added to the `global` namespace.

Reply:
The downstream changes for a global namespace still need to be handled. But for the scope of this PR, adding types which do not have a parent namespace should be good.